### ISSUE TITLE
Add ARIA attributes - Button

### ISSUE THREAD

**Issue by jrgarciadev** [nextui-org/nextui#38]
**Describe the bug**
Currently, the Button component doesn't have the ARIA attributes, this will have NextUI more accessible to people with disabilities.

**Expected behaviour**
Add Aria Attributes to components

**Additional info**
References:
[ARIA-Moz](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
[ARIA-Google](https://developers.google.com/web/fundamentals/accessibility/semantics-aria/)
[Chrome extension](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en)

**Comment by sauloviniciusf** [comment]
I will help with this too! 🙌🏼

**Comment by jrgarciadev** [comment]
For sure, Thanks! @sauloviniciusf  🙌🏼

**Comment by jrgarciadev** [comment]
Hi! @ibrahimtelman I hope you're doing well, do you have any updates on this issue? 🙋🏻‍♂️

**Comment by carlosriveroib** [comment]
If I understand correctly, is it only necessary to add the attributes aria-label with the value of children and aria-disabled with the value of the disabled prop to the component?

**Comment by jrgarciadev** [comment]
Hey @carlosriveroib eyes that's it and the 'tabindex=-1" when it's disabled and 'tabindex=0' when it's enabled, would you like to take this issue? 🙋🏻‍♂️

**Comment by carlosriveroib** [comment]
Yes, sure. What should I add to aria-label when there is only icon? Should I update docs with this info? Sorry this is going to be my first contribution to an open source project

**Comment by jrgarciadev** [comment]
Hey, @carlosriveroib it's not necessary to update the docs because this does not impact input/output parameters, regarding the button icon you could add  `role= presentation` to denote that the icon is not an interactive element, and only used for presentation 🙋🏻‍♂️

**Comment by carlosriveroib** [comment]
Pull request created!
